version.go

```diff
@@ -5,4 +5,4 @@
 package gin
 
 // Version is the current gin framework's version.
-const Version = "v1.4.0"
+const Version = "v1.4.1-dev"
```

Comment:
v1.4.0-dev

Reply:
@appleboy you mean `v1.5.0-dev`?

Reply:
@thinkerou I mean that `v1.4.0-dev` because of `git describe --tags --always`